### ISSUE TITLE
[HOLD for payment 2023-07-17] [$1000] Fix design for anonymous report footer 

### ISSUE THREAD

**Issue by marcochavezf** [Expensify/App#20699]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

Coming from this [comment](https://github.com/Expensify/App/issues/20494#issuecomment-1586691541), we want to fix the design and breakpoints for the report footer, which is displayed when an unauthenticated user opens a public room. Additionally, the report footer disappears in a room with several comments:

<img width="1510" alt="Screenshot 2023-06-13 at 11 28 08" src="https://github.com/Expensify/App/assets/6829422/21a89a26-aaf0-462c-bc35-c3cbdb9358ca">

## Action Performed:
Open a public room (like [ECX announce](https://staging.new.expensify.com/r/3504895439653267)) as unauthenticated user

## Expected Result:
The report footer should look like this:

![Chat - Desktop (5)](https://github.com/Expensify/App/assets/6829422/9f179a07-627a-40e0-ba2f-f79b3ac687d7)

![Chat (1)](https://github.com/Expensify/App/assets/6829422/9298e28c-4df8-45aa-9a6c-d97f21ac034d)

## Actual Result:
The breakpoints for mobile view are not displaying the elements as expected

![245055112-1d876fb7-df0a-467f-ad24-86576db768ed](https://github.com/Expensify/App/assets/6829422/4bb0a213-8107-4ddd-9786-aa9b22578ef9)

<img width="806" alt="245054715-fb3ea293-ad93-4a07-9c6a-e8deae7e8716" src="https://github.com/Expensify/App/assets/6829422/13002451-5fb6-4a8c-a8ff-2d1ec70fe10a">

And the footer should be fixed:

<img width="1510" alt="Screenshot 2023-06-13 at 11 28 08" src="https://github.com/Expensify/App/assets/6829422/21a89a26-aaf0-462c-bc35-c3cbdb9358ca">

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:**
**Reproducible in staging?:**
**Reproducible in production?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:**
**Issue reported by:**
**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01a1ba14253aff0dd0</li>
        <li>Upwork Job ID: 1668673550838931456</li>
        <li>Last Price Increase: 2023-06-13</li>
    </ul>
</details>

**Comment by Vishrut19** [comment]
Can I submit a proposal and work on this issue ?

**Comment by marcochavezf** [comment]
Since @allroundexperts asked to take this one on the other issue and implemented the report footer for anonymous users I think makes sense to assign him here

**Comment by Vishrut19** [comment]
> I don't know what kind off system this open source project has because you don't give chance to new folks to come and contribute to this project. There are set number of folks who just get all the issue, one after the other after the other.I think open source is a place where anyone can try to contribute and i always had the notion that it's all about collaboration, but now I got to know that it's all about some group of people only here in Expensify  

That's really awful from Expensify Team. Really Really Awful 👎👎

**Comment by mountiny** [comment]
@Vishrut19 sorry you having this experience. There are certain issues we prefer to offload to expert agencies we contract with in which case we assign those to their engineers but we try to make sure that in such case the issues dont have Help Wanted label to not waste contributors time.

There is plenty first time contributors you can see in the PR section, so I would definitely not say issues are assigned to only existing contributors, however, I appreciate its a steep learning curve to land your first PR. It is easier later on as you are more familiar with the codebase.

I hope you can still land your first and many PRs to come in future

**Comment by Vishrut19** [comment]
Also, I am unable get my hands on to the codebase and no one is ready to help. Is there any way I can get my hands on to the codebase ?

**Comment by mountiny** [comment]
You can see all the code here https://github.com/Expensify/App

**Comment by marcochavezf** [comment]
Hi @Vishrut19, my apologies. This is a follow-up of a previously implemented issue, and I decided to move with @allroundexperts since we want to fix it on production this week if possible, he implemented the component that we want to modify and offered to take this one first [here](https://github.com/Expensify/App/issues/20494#issuecomment-1589749355). Also, you can find [here](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) some of our guidelines to contribute to the codebase.

**Comment by marcochavezf** [comment]
@allroundexperts https://staging.new.expensify.com/r/3504895439653267

**Comment by marcochavezf** [comment]
Hi @shawnborton, can you do a design review for the screenshots/videos of this [PR](https://github.com/Expensify/App/pull/20884)?

**Comment by shawnborton** [comment]
Can do!

**Comment by marcochavezf** [comment]
PR in review. @shawnborton would you like to make a final review for the screenshots posted [here](https://github.com/Expensify/App/pull/20884#issuecomment-1606256155)?

**Comment by shawnborton** [comment]
Can do!

**Comment by muttmuure** [comment]
Last pass on the screenshots from @shawnborton when he gets back from the holiday weekend: 

https://github.com/Expensify/App/pull/20884#issuecomment-1614214756

**Comment by shawnborton** [comment]
Not overdue

**Comment by muttmuure** [comment]
Clarifying something [here](https://expensify.slack.com/archives/C01SKUP7QR0/p1689936524669259)

**Comment by muttmuure** [comment]
Catching up from ooo

**Comment by shawnborton** [comment]
Not overdue, just closing out payments here.

**Comment by marcochavezf** [comment]
> Not overdue, just closing out payments here.

**Comment by JmillsExpensify** [comment]
@muttmuure Can you summarize all payments on this issue before I approve in NewDot. More context [in Slack](https://expensify.slack.com/archives/C01SKUP7QR0/p1690321157273869).

**Comment by muttmuure** [comment]
@aimane-chnaif is right there is no penalty because it's 8 business days. Thanks for clarifying!

**Comment by muttmuure** [comment]
@aimane-chnaif - $1000 for C+
@allroundexperts - $1000 for C

**Comment by JmillsExpensify** [comment]
Reviewed details for @allroundexperts. These details are accurate based on summary from Business Reviewer and are now approved for payment in NewDot.

**Comment by muttmuure** [comment]
Catching up from OOO 

Offer sent to @aimane-chnaif

**Comment by muttmuure** [comment]
@aimane-chnaif paid, closing out
